src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -1312,15 +1313,29 @@ export const WorkflowView = _.flow(
           ]),
           exporting &&
             h(ExportWorkflowModal, {
-              thisWorkspace: workspace,
-              methodConfig: savedConfig,
+              defaultWorkflowName: savedConfig.name,
+              destinationWorkspace: ({ workspace: { workspaceId }, accessLevel }) => {
+                return workspace.workspaceId !== workspaceId && WorkspaceUtils.canWrite(accessLevel);
+              },
+              title: 'Copy to Workspace',
+              exportButtonText: 'Copy',
+              exportProvider: makeExportWorkflowFromWorkspaceProvider(workspace, savedConfig),
```

Comment:
same as in other usage, pop this make call out to be at module scope,  above component definition.

Reply:
(See comments on corresponding thread)